src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -237,8 +237,8 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
             projectContainer = getProjects(String(accessTokenField.password), settings.baseUri)
             with(projectBox) {
                 removeAllElements()
-                selectedItem = projectContainer.map { it.key }.firstOrNull()
-                projectContainer.map { addElement(it.key) }
+                selectedItem = projectContainer.map { it.key.plus(" (").plus(it.name).plus(")") }.firstOrNull()
+                projectContainer.map { addElement(it.key.plus(" (").plus(it.name).plus(")")) }
```

Comment:
```suggestion
                projectContainer.map { addElement(it.key + " (" + it.name + ")") }
```
this should achieve the same thing, and I think it's easier to read than `.plus(..)`

Reply:
I changed this to use a shared function, which itself uses string interpolation - does that address this concern?